### DISCUSSION TITLE
Limiting Forking/Cloning of a Public Repository

### DISCUSSION THREAD

**Discussion by OleksandrOleksyuk** [community/community#116723] [Repositories]
### Select Topic Area

Question

### Body

Hi everyone,
I'm considering making our repository public on GitHub, but I'd like to restrict forking or cloning of the code to authorized users. Does anyone have experience or suggestions on how we can implement this restriction? Thanks for your help!

**Comment by dannevesdantas** [comment]
I believe you can achieve it by creating a public organization and adding members to it.
